### PR TITLE
feat(camunda-platform): add script group

### LINKED ISSUE #47: Implement script props
* [ ] implement _Script_ group

image

**Note** that this props might also appear in other groups, e.g. Input.

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Closes #47

image

### CODE REVIEW

**Review by barmac — APPROVED** [review]
LGTM

**Review by andreasgeier — APPROVED** [review]
✅ &nbsp; LGTM

@pinussilvestrus As discussed, please create an issue to ensure default font-family style within textareas and set monospace explicitly and only for script textareas.

**Comment by pinussilvestrus** [comment]
I added an issue for that, thanks! 

https://github.com/bpmn-io/properties-panel/issues/87

File: src/provider/camunda-platform/properties/ScriptProps.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+  useService
+} from '../../../hooks';
+
+/**
+ * Cf. https://docs.camunda.org/manual/latest/user-guide/process-engine/scripting/
+ */
+export function ScriptProps(props) {
+  const {
+    element,
+    prefix
```
Comment (barmac) [inline src/provider/camunda-platform/properties/ScriptProps.js:19]: Do we pass prefix anywhere? I noticed this only after the PR has been merged.

Comment (pinussilvestrus) [reply src/provider/camunda-platform/properties/ScriptProps.js:19]: Currently, we don't use it. I was expecting that the `ScriptProps` is something we can re-use in some other places (listeners, input/output parameters, ...), at least some parts of it (single property components). Therefore the prefix would be needed to indicate different entries.

If we won't use it anywhere, we can safely remove it.

Comment (barmac) [reply src/provider/camunda-platform/properties/ScriptProps.js:19]: Well it doesn't hurt us either. Let's leave it as it is.

**Pull request merged**